pkg/test/framework/components/echo/echotest/filters.go

```diff
@@ -111,6 +111,11 @@ func isRegularPod(instance echo.Instance) bool {
 	return len(c.Subsets) == 1 && !c.IsVM() && !c.IsTProxy() && !c.IsNaked() && !c.IsHeadless()
 }
 
+// SpecialWorkloads includes VMs, external services, naked services, headless, etc.
+func SpecialWorkloads(instances echo.Instances) echo.Instances {
```

Comment:
nit: Could we just have the call site do this?

The name seems a bit awkward ... not that "regularPod" is better. Do we have the full classification of workload types? Maybe we should make it explicit in an enum or something?

Reply:
Yeah, `SpecialWorkloads` just means !regular, and the call site was doing `!special`.. so basically `regular`.